Makefile

```diff
@@ -83,6 +83,7 @@ DISABLED_HOST_VALIDATIONS := $(or ${DISABLED_HOST_VALIDATIONS}, "")
 DISABLED_STEPS := $(or ${DISABLED_STEPS}, "")
 DISABLE_TLS := $(or ${DISABLE_TLS},false)
 ENABLE_ORG_TENANCY := $(or ${ENABLE_ORG_TENANCY},False)
+ENABLE_ORG_BASED_FUNCTIONALITY := $(or ${ENABLE_ORG_BASED_FUNCTIONALITY},False)
```

Comment:
just a naming suggestion, maybe something like this could sound clearer: ENABLE_ORG_FEATURE_GATE

Reply:
I'm open to changing this. will do it later on in this PR.

Reply:
I agree with @danielerez that it's not clear what ENABLE_ORG_BASED_FUNCTIONALITY means - I think that it should be something like: ENABLE_ORG_CAPABILITY OR ENABLE_ORG_CAPABILITY_BY_AMS

Reply:
FEATURE_GATE makes it clear that it's used for enabling a new/experimental feature

Reply:
I changed it to `FEATURE_GATE`